public/app/plugins/datasource/prometheus/plugin.json

```diff
@@ -4,6 +4,11 @@
   "id": "prometheus",
   "category": "tsdb",
   "routes": [
+    {
+      "method": "GET",
+      "path": "api/v1/status/flags",
+      "reqRole": "Admin"
```

Comment:
Why do we want this to require admin? I know that in recorded queries we will only need this as an admin, but I'm not sure if it matters to require admin here. What are your thoughts?

Reply:
after talking with @MasslessParticle, it sounds like we will need to avoid prometheus specific status checks here so that it's easier to support prometheus compatible APIs. closing this one, and i'll take a different approach in a new PR